tracking/src/main/java/org/hps/recon/tracking/kalman/KalmanPatRecDriver.java

```diff
@@ -537,13 +517,7 @@ private ArrayList<KalTrack>[] prepareTrackCollections(EventHeader event, List<Tr
                 //Add the TrackResiduals
                 List<Integer> layers    = new ArrayList<Integer>();
 		List<Double> residuals  = new ArrayList<Double>();
-                List<Float> sigmas      = new ArrayList<Float>(); 
-		List<Integer> layersInt = new ArrayList<Integer>();
```

Comment:
Is there a reason why this chunk of code is moved down a few lines? It is hard to parse what is going on with the different loops and if statements.